mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -88,9 +91,10 @@
 
   private static final List<Type> SUPPORTED_TYPES =
           ImmutableList.of(Types.BooleanType.get(), Types.IntegerType.get(), Types.LongType.get(),
-                  Types.FloatType.get(), Types.DoubleType.get(), Types.DateType.get(), Types.TimestampType.withZone(),
+                  Types.FloatType.get(), Types.DoubleType.get(), Types.DateType.get(),
+                  Types.TimestampType.withZone(),
                   Types.TimestampType.withoutZone(), Types.StringType.get(), Types.BinaryType.get(),
-                  Types.DecimalType.of(3, 1), Types.UUIDType.get(), Types.FixedType.ofLength(5),
+                  Types.DecimalType.of(3, 1), Types.UUIDType.get(), Types.FixedType.ofLength(11),
```

Comment:
Can we write different decimal scales/precisions? If so, can we add a second one here, that has a different scale to test?

Reply:
I would rather not add a new entry to this list. If we would like to test the many flavours of the Decimal type, I guess we should do it in a dedicated method/class.